src/game.cpp

```diff
@@ -7218,12 +7219,13 @@ bool pet_menu(monster *z)
         drop_all,
         give_items,
         pheromone,
+        milk,
         rope
     };
 
     uimenu amenu;
     amenu.return_invalid = true;
-    std::string pet_name = _("dog");
+    std::string pet_name = _(z->get_name().c_str());
```

Comment:
`monster::get_name` returns a translated string, no need to translate it again (get rid of the `_()` and the `.c_str()`).